io.github.suchnsuch.Tangent.yaml

```diff
@@ -0,0 +1,71 @@
+id: io.github.suchnsuch.Tangent
+runtime: org.freedesktop.Platform
+runtime-version: '24.08'
+sdk: org.freedesktop.Sdk
+base: org.electronjs.Electron2.BaseApp
+base-version: '24.08'
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node22
+# Electron doesn't use a traditional locale format
+separate-locales: false
+writable-sdk: true
+command: run.sh
+finish-args:
+  - --device=dri
+  - --share=ipc
+  # Sticking to x.org for now
+  - --socket=x11
+  # Probably not needed
+  #- --socket=pulseaudio
+  # Needed to download dictionaries for spell-checking
+  - --share=network
+  # Uses portals, direct access is not needed
+  # Not including causes workspace selection to show up on every run
+  # Also trash doesn't work
+  #- --filesystem=home
+  # Not including system-bus access causes an error, but the app still works
+  #- --socket=system-bus
+  # Trash functionality doesn't seem to work no matter what I do
+  #- --filesystem=xdg-data/Trash
```

Comment:
```suggestion
  - --socket=x11
  - --share=network
```

Reply:
Is there any particular reason why these comments are inappropriate/unwanted?

Reply:
Put what you will use in the manifest, a bunch of comments and unused stuff makes it hard to read.

Reply:
Alright, understood.